src/main/java/com/xceptance/xlt/report/ReportGenerator.java

```diff
@@ -679,11 +689,52 @@ public void transformReport(final File inputXmlFile, final File outputDir) throw
             TaskManager.getInstance().waitForAllTasksToComplete();
             TaskManager.getInstance().stopProgress();
 
-            XltLogger.reportLogger.info(String.format("...finished - %,d ms", TimerUtils.get().getElapsedTime(start)));
+            XltLogger.reportLogger.info("...finished - {} ms", TimerUtils.get().getElapsedTime(start));
```

Comment:
Previously, we had a thousands separator in the milliseconds. This was convenient in case it takes a while to create the artifacts.

Reply:
Reverted change.